site/features/scalability.html

```diff
@@ -61,10 +61,10 @@ <h2>Containerized</h2>
         </p>
       </div>
       <div class="col-xs-12 col-lg-6 feature-card">
-        <img src="/assets/img/icons/lightweight-sessions.svg" alt="Lightweight Sessions">
-        <h2>Lightweight Sessions</h2>
+        <img src="/assets/img/icons/lightweight-sessions.svg" alt="Stateless UI">
+        <h2>Stateless User Interfaces</h2>
         <p>
-          While some components of FusionAuth use sessions and require session pinning, our sessions are lightweight. This means a single server can handle tens of thousands of sessions or more.
+          Each FusionAuth instance is completely stateless. Want to support more users? Add more nodes.
```

Comment:
"Technically", nodes does not have to increase as users increase, it is more about CPU performance, and login scale. 


What about: 

> As your user base grows, FusionAuth effortlessly scales to meet the login demands without interrupting existing  user sessions.

Reply:
I like the original text better actually. When I hear "effortlessly scales" it sounds automated and won't require new nodes. This is not really the case and it's definitely something people need to plan for and get DevOps involved in and all the things.